modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java

```diff
@@ -319,6 +319,8 @@ public void testUpdateDefaultObjectStateFlow() throws Exception {
 
 	@Test
 	public void testUpdateObjectStateTransitions() throws Exception {
+		_step4ListTypeEntry = _addListTypeEntry("step4");
```

Comment:
This code could be in the setUp() method?

Reply:
Perfect, i agree.